src/main/java/com/aws/greengrass/config/Topic.java

```diff
@@ -128,6 +128,10 @@ private Topic overrideValue(Object nv) {
         return withNewerValue(this.modtime, nv);
     }
 
+    private Topic overrideValue(Object nv, long proposedModTime) {
```

Comment:
This doesn't make sense, it won't do anything different from with newer value.
The point of overriding is that the value provided here is going to be set no matter what the timestamp is on the node. This implementation won't do that, this is a straight up alias for with newer value.